AGENTS.md

```
@@ -0,0 +1,26 @@
+# Git Commit Messages
+When generating git commit messages:
+- **ALWAYS use a prefix** (feat, fix, docs, style, refactor, perf, test, chore)
+- Keep the main message under 500 characters INCLUDING the prefix
+- Use imperative mood after the prefix
+- Be specific but concise
+- No periods at the end
+- Format: `prefix: message` and details should be written using bullet points.
+- **Language**: Always use English for commit messages
+- **Direction**: You are a senior developer. You never make oversights of the code changes. You need to check all code changes and set commit messages covering all the changes.
+
+### Prefix Rules (Angular Convention):
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)